README.md

```diff
@@ -19,7 +19,7 @@ tar xvf group_data.tar
 Add the hash_list file to Git:
 
 ```
-git add data/group-*/hash_list.txt
+git add group-*/hash_list.txt
```

Comment:
It takes practice, but try to make sure that you keep the pull request clean of any distracting changes.